docs/tutorials/end_to_end_example.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
<p>Change <code>vis_cpu</code> to <code>matvis</code>, add <code>fftvis</code> to the list of supported simulators, and maybe note that <code>healvis</code> is deprecated (or just remove it from the list)?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_sim/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Good catch. Changed to <code>matvis</code>,added <code>fftvis</code> and removed <code>healvis</code>.   </p>